CHANGELOG.md

```diff
@@ -44,6 +57,12 @@ catalog entries resulted in an error.
 - Bug in which `correct_cable_len` defaulted to `None` instead of `True`
 for `read_mwa_corr_fits`.
 
+### Fixed
+- Fixed a bug in `UVBase` where `allowed_failures` was being ignored if a parameter had
+`required=True` set.
+- Fixed a bug where selection and addition/concat methods did not operate correctly on
+flex-spw `UVCal` objects where `Nspws>1`.
+
```

Comment:
I think these should be added to the top of the above Fixed section rather than making a second Fixed section.

Reply:
Looks like this was a rebase gone awry -- merged the two fixed sections in 70f7fed6b2f12e6dfd1aff1c9e18d1627cd8c8b9